sdk/containerregistry/Microsoft.Azure.ContainerRegistry/src/Microsoft.Azure.ContainerRegistry.csproj

```diff
@@ -23,7 +23,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" VersionOverride="[5.1.2]" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" VersionOverride="5.1.2" />
```

Comment:
Do you require a minimum of 5.1.2? Can that be coded in the version here?

Reply:
@AlexGhiondea what do you mean? I think this specifies 5.1.2 as the min version...

https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges